src/orion/core/worker/experiment.py

```diff
@@ -471,3 +471,66 @@ def _is_different_from(self, config):
                 break
 
         return is_diff
+
+
+# pylint: disable=too-few-public-methods
+class ExperimentView(object):
+    """Non-writable view of an experiment
+
+    .. seealso::
+
+        :py:class:`orion.core.worker.experiment.Experiment` for writable experiments.
+
+    """
+
+    __slots__ = ('_experiment', )
+
+    #                     Attributes
+    valid_attributes = (["_id", "name", "refers", "metadata", "pool_size", "max_trials", "status"] +
+                        # Properties
+                        ["space", "algorithms", "stats"] +
+                        # Methods
+                        ["fetch_completed_trials"])
+
+    def __init__(self, name):
+        """Initialize viewed experiment object with primary key (:attr:`name`, :attr:`user`).
+
+        Try to find an entry in `Database` with such a key and config this object
+        from it import, if successful. Else, init with default/empty values and
```

Comment:
I feel like there is a missing period here, it should probably read :
> Try to find an entry in `Database` with such a key and configure this object from it. Import if succesful.

Reply:
Yes the sentence is terrible :sweat_smile: . I have commits in #92 for ExperimentView, I think I will import them in this PR. It will be better for the review.

Reply:
Yes please :)